src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1134,11 +1158,7 @@ CREATE TABLE transcription_workflow
 ,trs_id                      INTEGER                       NOT NULL  -- FK to transciption_status
 ,workflow_actor              INTEGER                       NOT NULL  -- FK to account_user
 ,workflow_ts                 TIMESTAMP WITH TIME ZONE      NOT NULL
-,workflow_comment            CHARACTER VARYING             
-,created_ts                  TIMESTAMP WITH TIME ZONE      NOT NULL
-,created_by                  INTEGER                       NOT NULL
-,last_modified_ts            TIMESTAMP WITH TIME ZONE      NOT NULL
-,last_modified_by            INTEGER                       NOT NULL
+--,workflow_comment            CHARACTER VARYING             

```

Comment:
Why do we have this commented out - have we settled on "remove workflow_comment from transcription_workflow" now, so that we need to start using `transcription_comment` table now in the services done so far?

Reply:
in v50, the primary key for transcription_workflow trw_id has been added as a foreign key to trascription_comment. this solves the potential for multiple comments to arise for a single transcription at a single workflow stage.
transcription_comment retains the foreign key tra_id to transcription, in the case that the comment is not workflow specific, or cannot be tied to a prior workflow stage in migrated data

Reply:
Yep, I can see the value in doing this - similar to having `trs_id` on the transcription_comment table as we suggested before. Regarding workflow_comment, it just means we have some more rework needed in our signed off services for option 2.x workflow (Requested and Rejected).
@barry-jones50410 Please feel free to update this branch with your latest scripts, so that we can merge v52 into Sprint 13 development.